sycl/test/array_param/array-kernel-param-nested-run.cpp

```diff
@@ -5,7 +5,6 @@
 // RUN: %CPU_RUN_PLACEHOLDER %t.out
 // RUN: %GPU_RUN_PLACEHOLDER %t.out
 // RUN: %ACC_RUN_PLACEHOLDER %t.out
-// XFAIL: *
```

Comment:
Is this change relevant to the patch?

Reply:
Yes, I think so. Recently we added support for array kernel arguments https://github.com/intel/llvm/pull/1841 . However some cases (nested arrays or struct array members) can be supported once we implement structs decomposing in this patch, so this test is enabled here.